feat(its): only canonical tokens can have remote interchain tokens (#261)
axelarnetwork/axelar-cgp-sui: .changeset/empty-jokes-agree.md

---
'@axelar-network/axelar-cgp-sui': patch
---

> add a check to interchain_token_service to ensure only canonical tokens can cause deploy_remote_interchain_token

Reviewer: ```suggestion
only allow canonical tokens to be deployed remotely
```
